tests/api2/test_005_interface.py

```diff
@@ -60,8 +63,17 @@ def get_payload(ws_client):
 
     return payload, to_validate
 
+# Make sure that our initial conditions are met
+def test_001_check_ipvx(request, ws_client, get_payload):
+    # Verify that dhclient is running
+    ps_count = int(SSH_TEST('ps -aux | grep dhclient | wc -l', user, password, truenas_server.ip)['stdout'])
```

Comment:
I wonder if it might be better to query interface API and see if we have dhcp activated for any interface there ?

Reply:
We could make the test a bit more robust by adding both checks. Ensure our API reflects that dhcp is started (using interface API) but also checking that dhclient is actually running is worthwhile as well. We should do both to cover all our bases.